Task50/Pages/TablePage.cs

```diff
@@ -0,0 +1,54 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Interactions;
+using OpenQA.Selenium.Support.UI;
+using System.Collections.Generic;
+
+namespace Task50
+{
+    class TablePage
+    {
+        private string url = "https://www.seleniumeasy.com/test/table-sort-search-demo.html";
+        private IWebDriver driver;
+
+        public TablePage(IWebDriver driver)
+        {
+            this.driver = driver;
+            this.driver.Navigate().GoToUrl(url);
+        }
+
+        public void SelectDropdownItem()
+        {
+            SelectElement dropdown = new SelectElement(driver.FindElement(By.Name("example_length")));
+            dropdown.SelectByValue("10");
```

Comment:
I would recommend to add method parameter there instead of straight passing constant value